packages/orchestration/src/orchestration-api.ts

```diff
@@ -23,8 +23,31 @@ import type {
   NobleMethods,
   Bech32Address,
 } from './types.js';
+
 import type { ResolvedContinuingOfferResult } from './utils/zoe-tools.js';
 
+export type OrchestrationOptions = {
+  _placeholder?: never;
+};
+
+export type PacketOptions = OrchestrationOptions & {
+  sendOpts?: SendOptions;
+};
+
+/**
+ * Options relevant to all queries (non-mutating).
+ */
+export type QueryOptions = PacketOptions & {
+  queryOpts?: Record<string, any>;
+};
```

Comment:
very confusing that the QueryOptions type has a `queryOpts` property. I would expect `queryOpts?: QueryOptions`.

perhaps this is `QueryFnOpts` which has `queryOpts` for options on the querying it does?

Reply:
`QuerierOptions` sgtm. (`QueryFnOpts` has some appeal but the other option sets are for functions too and don't have 'Fn')